know/rdf/tests/wikidata_tests.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+from rdf.parser.sparql_reader import read_sparql
+
+
+class ReadTest(unittest.TestCase):
+    """ Sample test case """
+    def setUp(self):
+        # You can optionally setup things here that can be used in the rest of the tests
+        pass
+
```

Comment:
```suggestion
```
not needed so we can remove it - I only had it in `sample_tests.py` just to show y'all in case y'all needed it.